presto-memory/src/test/java/com/facebook/presto/plugin/memory/TestMemorySmoke.java

```diff
@@ -101,47 +97,46 @@ public void testCreateFilteredOutTable()
     public void testSelectFromEmptyTable()
             throws SQLException
     {
-        queryRunner.execute("CREATE TABLE test_select_empty AS SELECT * FROM tpch.tiny.nation WHERE nationkey > 1000");
+        computeActual("CREATE TABLE test_select_empty AS SELECT * FROM tpch.tiny.nation WHERE nationkey > 1000");
 
         assertQueryResult("SELECT count(*) FROM test_select_empty", 0L);
     }
 
     @Test
     public void testSelectSingleRow()
     {
-        assertQuery("SELECT * FROM nation WHERE nationkey = 1", "SELECT * FROM tpch.tiny.nation WHERE nationkey = 1");
+        assertQueryEquals("SELECT * FROM nation WHERE nationkey = 1", "SELECT * FROM tpch.tiny.nation WHERE nationkey = 1");
     }
 
     @Test
     public void testSelectColumnsSubset()
             throws SQLException
     {
-        assertQuery("SELECT nationkey, regionkey FROM nation ORDER BY nationkey", "SELECT nationkey, regionkey FROM tpch.tiny.nation ORDER BY nationkey");
+        assertQueryEquals("SELECT nationkey, regionkey FROM nation ORDER BY nationkey", "SELECT nationkey, regionkey FROM tpch.tiny.nation ORDER BY nationkey");
     }
 
     @Test
     public void testCreateTableInNonDefaultSchema()
     {
-        queryRunner.execute(format("CREATE SCHEMA %s.schema1", CATALOG));
-        queryRunner.execute(format("CREATE SCHEMA %s.schema2", CATALOG));
+        computeActual(format("CREATE SCHEMA %s.schema1", CATALOG));
+        computeActual(format("CREATE SCHEMA %s.schema2", CATALOG));
 
         assertQueryResult(format("SHOW SCHEMAS FROM %s", CATALOG), "default", "information_schema", "schema1", "schema2");
-
-        queryRunner.execute(format("CREATE TABLE %s.schema1.nation AS SELECT * FROM tpch.tiny.nation WHERE nationkey %% 2 = 0", CATALOG));
-        queryRunner.execute(format("CREATE TABLE %s.schema2.nation AS SELECT * FROM tpch.tiny.nation WHERE nationkey %% 2 = 1", CATALOG));
+        computeActual(format("CREATE TABLE %s.schema1.nation AS SELECT * FROM tpch.tiny.nation WHERE nationkey %% 2 = 0", CATALOG));
+        computeActual(format("CREATE TABLE %s.schema2.nation AS SELECT * FROM tpch.tiny.nation WHERE nationkey %% 2 = 1", CATALOG));
 
         assertQueryResult(format("SELECT count(*) FROM %s.schema1.nation", CATALOG), 13L);
         assertQueryResult(format("SELECT count(*) FROM %s.schema2.nation", CATALOG), 12L);
     }
 
     private List<QualifiedObjectName> listMemoryTables()
     {
-        return queryRunner.listTables(queryRunner.getDefaultSession(), "memory", "default");
+        return getQueryRunner().listTables(getSession(), "memory", "default");
     }
 
     private void assertQueryResult(String sql, Object... expected)
```

Comment:
Annotate the `sql` parameter with `@Language("SQL")` so that IntelliJ can track it all the way down